client/src/util/auth/Authentication.ts

```diff
@@ -28,6 +40,22 @@ export async function signOut() {
     const idToken = auth.user.id_token; // camelCase for variable name
     localStorage.clear();
     sessionStorage.clear();
+    clearCookies();
+
+    // Sign out silently
+    await auth.signoutSilent();
+
+    // Revoke tokens
+    await auth.revokeTokens();
+
+    // Make HTTP GET call to logout URL
+    await fetch(`${process.env.REACT_APP_URL}/_oauth/logout`, {
```

Comment:
Some of the installation scenarios don't have a server listening at this address. If not the `fetch` might throw an error. You might want to write a promise with enough code to handle the exceptions. There might be some in the `fetch` as well. Please check.